bacom/fix-breadcrumb-link.js

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { writeFile } from 'fs/promises';
+import { mdast2docx } from '@adobe/helix-md2docx';
+import { mkdir } from 'fs'; 
+import {
+    fetchText,
+    getMdast,
+    getTable,
+  } from '../utils/mdast-utils.js';
+
+async function setEntries(entry) {
+    const entries = await import(`./path-files/${entry}`, {assert: {type: 'json'}});
+
+    return entries.default.paths;
+}
+
+function getLocaleFromEntry(entry) {
+    return entry.split('-')[0];
+}
+
+/**
+ * A variation of the "getLeaf" function. Removes the second conditional check for
+ * "!node.children", which was returning the incorrect item
+ * 
+ * 
+ * @param {object} node / mdast json object with children
+ * @param {stirng} type / a type, like "link"
+ * @returns 
+ */
+const getLink = (node, type) => {
+    if (node?.type === type) return node;
+  
+    if (node.children) {
+      for (let i = 0; i < node.children.length; i++) {
+        const leaf = getLink(node.children[i], type);
```

Comment:
If it has a child then it's not a leaf node, right? Should the variable name be changed?